Hybrid/example.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+require 'azure_mgmt_resources'
+
+LOCAL = 'local'
+GROUP_NAME = 'azure-sample-group'
+
+# Manage resources and resource groups - create, update and delete a resource group, deploy a solution into a resource
+#   group, export an ARM template. Create, read, update and delete a resource
+#
+# This script expects that the following environment vars are set:
+#
+# AZURE_TENANT_ID: with your Azure Active Directory tenant id or domain
+# AZURE_CLIENT_ID: with your Azure Active Directory Application Client ID
+# AZURE_CLIENT_SECRET: with your Azure Active Directory Application Secret
+# AZURE_SUBSCRIPTION_ID: with your Azure Subscription Id
+#
+def run_example
+  #
+  # Create the Resource Manager Client with an Application (service principal) token provider
+  #
+  subscription_id = ENV['AZURE_SUBSCRIPTION_ID'] || '11111111-1111-1111-1111-111111111111'
+  active_directory_settings = get_active_directory_settings()
+
+   # your Azure Subscription Id
+  provider = MsRestAzure::ApplicationTokenProvider.new(
+      ENV['AZURE_TENANT_ID'],
+      ENV['AZURE_CLIENT_ID'],
+      ENV['AZURE_CLIENT_SECRET'],
+      active_directory_settings
+      )
+  credentials = MsRest::TokenCredentials.new(provider)
+
+  options = {
+      credentials: credentials,
+      subscription_id: subscription_id,
+      active_directory_settings: active_directory_settings,
+      base_url: 'https://management.<location>.azurestack.external'
```

Comment:
Example should be runnable as such against ASDK one node deployment. I think we should hardcode the location to be local in the example.rb

Reply:
or base url could be a environment var just like tenant id

Reply:
this comment is applicable for other samples as well